docs/workspaces.md

```diff
@@ -601,6 +597,26 @@ the storage solution that you are using.
 * `ReadWriteMany` is the least commonly available Access Mode. If you use this access mode and these volumes are available
   to all Nodes within your cluster, you may want to disable the Affinity Assistant.
 
+### Persistent Volume Availability Zone
+
+`Persistent Volumes` are usually "zonal" (i.e. they live within a single Availability Zone and cannot be accessed from a `pod` living in another Availability Zone). When using a workspace backed by a `PersistentVolumeClaim` (typically only available within a Data Center), the `TaskRun`
+`pods` can be scheduled to any Availability Zone in a regional cluster. This leads to two kinds of potential Availability Zone scheduling conflict.
+To avoid such conflict in `PipelineRuns`, Tekton provides [Affinity Assistants](affinityassistants.md) that schedule all `PipelineTaskRun` Pods or all `PipelineTaskRun` sharing a `PersistentVolumeClaim` to the same Node depending on the `coschedule` mode. 
+
+#### `PipelineTaskRuns` sharing the same `PersistentVolumeClaim` are scheduled to different Availability Zones.
+If two `PipelineTaskRuns` sharing the same `PersistentVolumeClaim` based workspace are scheduled to different Availability Zones, it is impossible to 
+schedule a `PersistentVolume` that can be accessed by both `PipelineTaskRuns`.
+
+The Affinity Assistant provides `coschedule: workspaces` mode to enforce scheduling all the `PipelineTaskRuns` `pods` sharing the same `PersistentVolumeClaim` based workspace to the same node (Availability Zone). By scheduling the `PersistentVolume` to the same Availability Zone as the `pods`, the deadlock is avoided.
+
+#### Multiple `PersistentVolumeClaims` in different Availability Zones are binded to a `PipelineTaskRun`
+If multiple `PersistentVolumeClaims` are binded to a `PipelineTaskRun` and the `PersistentVolumeClaims` are scheduled to different Availability Zone,
+it is impossible to schedule the `PipelineTaskRun` to an Availability Zone that can access all the binded `PersistentVolumeClaims` (i.e. `pvc1` is scheduled to Zone 1 and `pvc2` is scheduled to Zone 2, where should I schedule the `PipelineTaskRun`?)
```

Comment:
This docs section is a bit confusing; what is intended to be the distinction between this docs section and the previous one? It also sounds like scheduling is a choice that the user is making rather than how the k8s cluster handles taskrun pods. Can you reframe to focus a bit more on the user perspective, maybe including an example pipelinerun?